src/tasks/cell_cell_communication_source_target/api/file_solution.yaml

```diff
@@ -0,0 +1,62 @@
+type: file
+example: "resources_test/cell_cell_communication_source_target/pancreas/solution.h5ad"
+info:
+  label: "Solution"
+  summary: "The solution for the test data."
+  slots:
+    layers: 
+      - type: integer
+        name: counts
+        description: Raw counts
+        required: true
+      - type: double
+        name: normalized
+        description: Normalized expression values
+        required: true
+    obs:
+      - type: string
+        name: label
+        description: Cell type annotation.
+        required: true
+    var:
+      - type: double
+        name: hvg_score
+        description: High variability gene score (normalized dispersion). The greater, the more variable.
+        required: true
+    uns:
+      - type: string
+        name: dataset_id
+        description: "A unique identifier for the dataset"
+        required: true
+      - name: dataset_name
+        type: string
+        description: Nicely formatted name.
+        required: true
+      - type: string
+        name: dataset_url
+        description: Link to the original source of the dataset.
+        required: false
+      - name: dataset_reference
+        type: string
+        description: Bibtex reference of the paper in which the dataset was published.
+        required: false
+      - name: dataset_summary
+        type: string
+        description: Short description of the dataset.
+        required: true
+      - name: dataset_description
+        type: string
+        description: Long description of the dataset.
+        required: true
+      - name: dataset_organism
+        type: string
+        description: The organism of the sample in the dataset.
+        required: false
+      - type: string
+        name: ccc_target
+        description: assumed truth value. A pandas data frame containing response, source, and target
+        required: true
```

Comment:
* Please detail exactly what information is inside the data frame and what it represents. For example:
  - `source`: The cell type that initiates the communication signal.
  - `target`: The cell type which is intended to receive the communicational signal.
  - `ligand`: The gene of the signalling molecule released by the source cell.
  - `receptor`: The gene of the protein on the surface of the target cell that the ligand specifically binds to.
* This is not a string. Perhaps `type: list` or `type: dataframe` is more suitable. This, however, is the first case we need to store a data frame in the uns, so we'll have to add this structure. **Alternatively**: it would be possible to simply add the different columns of the `ccc_target` directly to the `uns` itself, at least for now.

what do you think of something like this?
```yaml
      - type: string
        name: ccc_source
        description: The cell type that initiates the communication signal. This serves as the ground-truth cell-cell communication (CCC) information. This vector contains one value per CCC interaction.
        required: true
      - type: string
        name: ccc_target
        description: The cell type which is intended to receive the communicational signal. This vector needs to be of the same length as `ccc_source`.
        required: true
      - type: string
        name: ccc_ligand
        description: The gene of the signalling molecule released by the source cell. This vector needs to be of the same length as `ccc_source`.
        required: true
      - type: string
        name: ccc_receptor
        description: The gene of the protein on the surface of the target cell that the ligand specifically binds to. This vector needs to be of the same length as `ccc_source`.
        required: true
```

Alternatively, something like this would also be possible, but then we need to add this data type to the openproblems_v2 helper functions:

```yaml
      - type: dataframe
        name: ccc_groundtruth
        description: A set of cell-cell communication (CCC) interactions. This data frame contains one row per CCC interaction.
        columns:
          - type: string
            name: source
            description: The cell type that initiates the communication signal.
            required: true
          - type: string
            name: target
            description: The cell type which is intended to receive the communicational signal.
            required: true
          - type: string
            name: ligand
            description: The gene of the signalling molecule released by the source cell.
            required: true
          - type: string
            name: receptor
            description: The gene of the protein on the surface of the target cell that the ligand specifically binds to.
            required: true
```

I think the second option would be the best, but again, some code in the `src/common/` folder would need to be updated for this to function as intended.

What do you think?

Reply:
Hey Robrecht! 
Sure, yeah i can see that this will be the first of its type as being a df. Even i agree that the second choice looks quite neater and like a best fit. If you think updating the code in the `src/common` directory wont be complicated, we can go for this. Can you tell me what has to be edited over there?

Reply:
Could you start off with the first approach for the time being? Because I'm away for little over a week :) Let's keep the `type: dataframe` in mind for when I get back.

Reply:
Sure!